### PR TITLE
@sweir27 => Auction blocks responsive styles

### PR DESCRIPTION
**Pull request by eessex**
- Adds responsive/mobile layout to auction-blocks component
- Fixes some key/propTypes errors

screen shot 2017-08-18 at 1 43 38 pm

### CODE REVIEW

File: desktop/apps/auctions2/components/Layout.js [hunk 1]
```diff
@@ -6,21 +6,17 @@ import { connect } from 'react-redux'
...
-          {auctions.timed.length &&
-            <AuctionsBlock
-              key='auctions-block-timed'
-              auctions={auctions.timed} /> }
-          </div> }
+      {auctions.liveIntegration.length &&
+        <AuctionsBlock
+          key='auctions-block-liveIntegration'
+          auctions={auctions.liveIntegration}
+          liveIntegration={true}
```
Comment (damassi) [inline desktop/apps/auctions2/components/Layout.js]: @eessex - If a prop on a react component is truthy you can omit the value, e.g., `<AuctionsBlock liveIntegration />`

File: desktop/apps/auctions2/components/auctions_block/AuctionsBlock.styl [hunk 2]
```diff
@@ -33,5 +33,36 @@
...
-    .date
-      avant-garde-size s-headline
+    display flex
+    flex-direction column
+    justify-content space-between
+    height 100%
+  .date
+    avant-garde-size s-headline
+
+@media (max-width: 1250px)
```
Comment (damassi) [inline desktop/apps/auctions2/components/auctions_block/AuctionsBlock.styl]: For responsive styles, this [mixin function](https://github.com/artsy/force/blob/master/desktop/components/stylus_lib/index.styl#L261) was recently added that allows one to do 

```css
.some-class
  background: black

  +respond-mobile() 
    background: green
  
  +respond-desktop()
    background: blue
  
```

Comment (eessex) [inline desktop/apps/auctions2/components/auctions_block/AuctionsBlock.styl]: Nice!

**Comment by sweir27** [comment]
👍 many thanks!!

**Pull request merged**